tests/compatibility-test.py

```diff
@@ -177,13 +178,7 @@ def test_kill_head(self):
         rtn = shell_subprocess_run(
                 'kubectl wait --for=condition=ready pod -l rayCluster=raycluster-compatibility-test --all --timeout=900s', check = False)
         if rtn != 0:
-            shell_subprocess_run('kubectl get pods -A')
-            shell_subprocess_run(
-                'kubectl describe pod $(kubectl get pods | grep -e "-head" | awk "{print \$1}")')
-            shell_subprocess_run(
-                'kubectl logs $(kubectl get pods | grep -e "-head" | awk "{print \$1}")')
-            shell_subprocess_run(
-                'kubectl logs -n $(kubectl get pods -A | grep -e "-operator" | awk \'{print $1 "  " $2}\')')
+            show_cluster_info("default")
         assert rtn == 0
```

Comment:
Remove the assert and raise an exception in the `if` statement?

Reply:
Updated https://github.com/ray-project/kuberay/pull/775/commits/7d640b0e8cacefef658aed7ee1cabf6431a3eb3d.